AstarteDeviceSDKCSharp/Data/AstarteDbContext.cs

```diff
@@ -42,7 +42,7 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
 
             if (!optionsBuilder.IsConfigured)
             {
-                optionsBuilder.UseSqlite("Filename = " + _persistencyDir + "\\AstarteDeviceDb", option =>
+                optionsBuilder.UseSqlite("Filename = " + _persistencyDir + "/AstarteDeviceDb", option =>
```

Comment:
Use [Path.DirectorySeparatorChar](https://learn.microsoft.com/en-us/dotnet/api/system.io.path.directoryseparatorchar?view=net-6.0), it provides a platform-specific character used to separate directory levels in a path string that reflects a hierarchical file system organization. 
Avoid to use explicit linux path.